src/modules/sidebar/mixins/shareAPI.js

```diff
@@ -33,50 +28,27 @@ export default {
 					permissionDelete: false,
 					permissionManage: false,
 				}
-				// console.debug('data array', data)
 				const res = await axios.post(generateUrl('/apps/tables/share'), data)
-				if (res.status === 200) {
-					await this.$store.dispatch('setTableHasShares', { tableId: res.data.nodeId, hasSHares: true })
-				} else {
-					showWarning(t('tables', 'Sorry, something went wrong.'))
-					console.debug('axios error', res)
-					return false
-				}
-				// console.debug('new share was saved', res)
+				await this.$store.dispatch('setTableHasShares', { tableId: res.data.nodeId, hasSHares: true })
```

Comment:
Any exception triggered in this line will also be handled by the `catch` block. In this case the request was successful. The share was created on the server but the update of the local state failed. However the UI will still display `Could not create share`.

But why would a dispatch fail you say...
Maybe due to a typo :wink: 
```suggestion
				await this.$store.dispatch('setTableHasShares', { tableId: res.data.nodeId, hasShares: true })
```

This typo will most likely not cause an exception to be thrown... I'd still recommend keeping the `try` block as small as possible, `throw` or `return` at the end of the `catch` block and move the update of the ui after the `catch` block to make sure errors during the ui update are not misinterpreted as request errors.